### PR TITLE
proposed improvement

### PR DESCRIPTION
**Pull request by bwhm**
Creating this issue to improve the up until now neglected repo.

### CODE REVIEW

**Comment by bwhm** [comment]
More info [here](https://testnet.joystream.org/#/proposals/30)

File: README.md [hunk 1]
```diff
@@ -1,4 +1,75 @@
...
+If a KPI requires submitting a deliverable, eg. reports or some code, it is expected that a PR is made to this repo in order to qualify.
+
+Although the community is meant to control the repo, Jsgenesis will approve and merge any pull requests for now. Note that the repo is licensed under [GPLv3](/LICENSE).
+
+## Workflow
+
+The workflow for changing the repo depends on the reason and purpose behind the change.
+A consistent part is for the contributor to fork the repo, and create a pull request to the applicable branch.
+
+### KPI Related Submittals
```
Comment (blrhc) [inline README.md:49]:
```suggestion
### KPI Related Submissions
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,75 @@
...
+When a KPI requires a deliverable to be successful, the following steps must be made:
+- A pull request is made to the master branch.
+- A proposal is made to the [Joystream testnet](https://testnet.joystream.org/).
+  - The proposal (`Text`, or in some cases, `Spending`) contains a link to the PR and other relevant information
+  - When (if) the proposal is voted through, @bwhm and @blrhc is tagged
+  - The time of the latest commit will be used as the time of submission
+- The PR is reviewed, and as long as it does not contain anything malicious or does not comply with license, it is merged.
+
+### Individual Submittals
```
Comment (blrhc) [inline README.md:58]:
```suggestion
### Individual Submissions
```

File: README.md [hunk 3]
```diff
@@ -1,4 +1,75 @@
...
+**Placeholder: ->**
+If the deliverable is made by an individual, eg. for an existing or upcoming funding proposal, the following steps must be made:
+- A pull request is made to the community branch?
+- Action?
+- Merged to community branch?
+- PR to master
+- The PR is reviewed, and as long as it does not contain anything malicious or does not comply with license, it is merged.
+**<- Placeholder**
+
+### Jsgenesis Submittals
```
Comment (blrhc) [inline README.md:68]:
```suggestion
### Jsgenesis Submissions
```

File: README.md [hunk 4]
```diff
@@ -1,4 +1,75 @@
...
+- The PR is reviewed, and as long as it does not contain anything malicious or does not comply with license, it is merged.
+
+### Individual Submittals
+**Placeholder: ->**
+If the deliverable is made by an individual, eg. for an existing or upcoming funding proposal, the following steps must be made:
+- A pull request is made to the community branch?
+- Action?
+- Merged to community branch?
+- PR to master
+- The PR is reviewed, and as long as it does not contain anything malicious or does not comply with license, it is merged.
```
Comment (blrhc) [inline README.md:65]:
```suggestion
- The PR is reviewed, and as long as it does not contain anything malicious or does not comply with the license of the repo, it is merged.
```

File: README.md [hunk 5]
```diff
@@ -1,4 +1,75 @@
...
+- [Overview](#overview)
+  - [Workflow](#workflow)
+    - [KPI Related Submittals](#kpi-related-submittals)
+    - [Individual Submittals](#individual-submittals)
+    - [Jsgenesis Submittals](#jsgenesis-submittals)
+<!-- TOC END -->
+
+# Overview
+
+The Joystream Community Repo is meant both as a resource for the community members of the Joystream project, and a place to submit their work or contributions.
```
Comment (blrhc) [inline README.md:38]:
```suggestion
The Joystream Community Repo is intended both as a resource for the community members of the Joystream project, and a place to submit their work or contributions.
```

File: README.md [hunk 6]
```diff
@@ -1,4 +1,75 @@
...
+- A pull request is made to the community branch?
+- Action?
+- Merged to community branch?
+- PR to master
+- The PR is reviewed, and as long as it does not contain anything malicious or does not comply with license, it is merged.
+**<- Placeholder**
+
+### Jsgenesis Submittals
+**Placeholder: ->**
+If a member Jsgenesis wants to make changes to the repo, the following steps must be made:
```
Comment (blrhc) [inline README.md:70]:
```suggestion
If a member of the Jsgenesis team wants to make changes to the repo, the following steps must be taken:
```

**Comment by bwhm** [comment]
@blrhc 
Unless you see anything bad, please merge!

**Pull request merged**